packages/deployer/test/helpers/eventSystem.js

```diff
@@ -24,12 +24,18 @@ const postDeployOccurredForNames = (options, contractNames) => {
   }, true);
 };
 
-const linkingOccurredForName = (options, contractName, libraryName) => {
+const linkingOccurred = (
```

Comment:
Is there any way to get this to distinguish the type of linking that occurred?

Reply:
What do you mean?  Like instance vs class?  That'd have to be emitted in the event, and right now the event only gives names and addresses.  Is that information we want to put in the event?  It's not clear to me what a reporter would do with that, or why we'd want to check that here... what matters ultimately is the address.

Reply:
I think the interface distinction matters because it could expose a different intention by the user. It is something I would want to see in the migration log/reporter, preferring to err on the side of presenting more information to the user. Perhaps I'm wrong, and I also don't want to hold your PR back. We can always revisit what we present later on if it becomes an issue

Reply:
Hm, I wonder if other people have opinions on this, regarding if this is something that should be added, if the reporter should display this... @gnidan? @eggplantzzz? @lsqproduction? @sukanyaparashar?

(And yeah I'll probably just merge this later, but there's no hurry; nothing depends on this and the release is days away, so I'm OK with waiting a bit longer while people discuss. :) )

Reply:
Offhand I can't really think of something useful to print about this. How would you word it @cds-amal? It seems pretty unhelpful to me to say something like "Linking class instance" versus something else. What kind of different intention might the  user have? I'm kind of with @haltman-at on this one. But if you have a more specific idea of what and how to implement this I'm all ears.

Reply:
OK since nobody else has responded I'm going to go ahead and merge this shortly, if you want to respond do so quickly!